README.md

```diff
@@ -1,7 +1,5 @@
 # Librarian-puppet
 
-[![Build Status](https://travis-ci.org/voxpupuli/librarian-puppet.png?branch=master)](https://travis-ci.org/voxpupuli/librarian-puppet)
```

Comment:
do you want to add links for the new github action status?

Reply:
Personally I'd rather not.